src/main/java/core/basesyntax/service/date/DateConverterImpl.java

```
@@ -0,0 +1,62 @@
+package core.basesyntax.service.date;
+
+import core.basesyntax.service.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DateConverterImpl implements DateConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> rawData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+
+        if (rawData == null || rawData.isEmpty()) {
+            return transactions;
+        }
+
+        for (String line: rawData) {
+
+            if (line.startsWith("type,fruit,quantity")) {
+                continue;
+            }
+
+            String[] parts = line.split(",");
```

It's not a good idea to continue the loop if the line does not contain exactly 3 parts. It would be better to throw an exception in this case to let the user know that the input data is not valid.